src/app/handlers/auto_unverify.handler.ts

```diff
@@ -0,0 +1,18 @@
+import { IContainer, IHandler, IMessage } from '../../common/types';
+
+export class AutoUnverifyHandler implements IHandler {
+  constructor(public container: IContainer) {}
+
+  public async execute(message: IMessage): Promise<void> {
+    const avatar = message.author.avatar; //This is null if they have default avatar
```

Comment:
this can have a more descriptive name so we can remove the comment

`const hasAvatar = Boolean(message.author.avatar);`